packages/e2e/src/e2e.test.ts

```diff
@@ -141,6 +141,10 @@ describe('CLI', () => {
   });
 
   describe('dxns', () => {
+    it('create dxns account', async () => {
+      await cmd('dxns account recover --mnemonic "//Alice"')
+    })
+
     it('seed', async () => {
```

Comment:
can you add a test for adding devices here?

Reply:
✅